angr/analyses/cfg/cfg_fast.py

```diff
@@ -1780,16 +1779,14 @@ def _create_jobs(self, target, jumpkind, current_function_addr, irsb, addr, cfg_
         jobs = [ ]
 
         if target_addr is None and (
-                        jumpkind in ('Ijk_Boring', 'Ijk_Call') or jumpkind.startswith('Ijk_Sys'))\
-                and fast_indirect_jump_resolution:
+                        jumpkind in ('Ijk_Boring', 'Ijk_Call') or jumpkind.startswith('Ijk_Sys')):
```

Comment:
Here we still want to keep the `fast_indirect_jump_resolution=True` check so there is a way to disable this functionality when calling `_create_jobs()` -- although it is not used currently.

Reply:
I think we could just use self._resolve_indirect_jumps?

Reply:
Sure.